### PR TITLE
LRAC-8289 | Create test stubs for SiteInterests

### PR DESCRIPTION
**Pull request by LucasAcacio**

### CODE REVIEW

**Review by alexcandidos — COMMENTED** [review]
Hi @LucasAcacio , 

the description should be like:

`   @description = "Feature ID: Legacy | Automation ID: LRAC-8119 | Test Summary: Create Access Request`

it is:

 @description = "**Automation ID**: Legacy | **Automation ID**: LRAC-8119 | Test Summary: Create Access Request

**Comment by alexcandidos** [comment]
Could you please review?

Thanks!

**Comment by LucasAcacio** [comment]
Required changes done

**Comment by ViniLopes20** [comment]
aa

**Comment by alexcandidos** [comment]
Hi @LucasAcacio , Thanks for working on it!

The code looks good to me!

**Comment by alexcandidos** [comment]
ci:forward

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SitesInterests.testcase [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+		ACUtils.tearDownAC();
+	}
+
+	@description = "Feature ID: Legacy | Automation ID: LRAC-8287 | Test Summary: Assert Sites Interests card highest interests"
+	@ignore = "true"
+	@priority = "3"
+	test AssertHigherInterests {
+
+		// TODO LRAC-8287 CanCreateAccessRequest
+
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SitesInterests.testcase:38]: hi @LucasAcacio , it should be AssertHigherInterests

**Pull request closed without merging**